languages/rust/exercises/concept/tuples/.meta/design.md

```diff
@@ -0,0 +1,34 @@
+## Learning Objectives
+
+After completing the exercise, the user should:
+
+- know how to create tuples
+- understand tuple access syntax `a_tuple.0`
+- know how to destructure tuples into new variables, e.g. `let (x, y) = a_point;`
+
+## Out of scope
+
+- tuple structs
+- unit structs
+- enums with tuple structs
+
+## Concepts
+
+The Concepts this exercise unlocks are:
+
+- tuples
+- destructuring
+
+## Prequisites
+
+This exercise's prerequisites Concepts are:
+
+- structs
+
+## Representer
+
+No changes required
+
+## Analyzer
+
+No changes required
```

Comment:
For the C# track, I opted to remove these sections if there is no data in them (like in this case). Feel free to do this if you think that makes sense.